### DISCUSSION TITLE
Once a month meetings

### DISCUSSION THREAD

**Discussion by eflashlogistics** [code4recovery/12-step-meeting-list#1326] [Q&A] [answered]
Hello! Do we have a way to list meetings only once a month? Our speaker meeting is only once a month. Thank you!

**Comment by SIA-WEBSUP** [comment, marked as answer]
@eflashlogistics , the TSML plug-in is geared towards weekly AA meetings. Normally AA entities publish their monthly service meetings or a speaker exchanges on separate page from the AA meetings and not using the plugin.

Please see the following thread for more info: https://github.com/code4recovery/12-step-meeting-list/discussions/919
